e2e/testsuite/testsuite.go

```diff
@@ -29,13 +36,18 @@ const (
 // E2ETestSuite has methods and functionality which can be shared among all test suites.
 type E2ETestSuite struct {
 	suite.Suite
+	grpcClientSets map[string]GRPCClientSet
 	paths          map[string]path
 	logger         *zap.Logger
 	DockerClient   *dockerclient.Client
 	network        string
 	startRelayerFn func(relayer ibc.Relayer)
 }
 
+type GRPCClientSet struct {
```

Comment:
nit: I think we should add godocs to all exported types

Reply:
What's the reasoning behind the naming of having the `Set` suffix? Initially when I read this I thought it was supposed to be a set data structure but I'm slightly confused

Reply:
Yep sounds good to me! :) Thanks 🤝